src/types/formatted/pressRelease.ts

```diff
@@ -0,0 +1,18 @@
+import { PublishedEntity } from './publishedEntity'
+import { NodeOffice, NodePersonProfile, NodeEventListing } from '../drupal/node'
+import { DrupalMediaDocument, DrupalMediaImage, DrupalMediaVideo } from '../drupal/media'
+import { FieldAddress, FieldFormattedText } from '../drupal/field_type'
+
+export type PressRelease = PublishedEntity & {
```

Comment:
these formatted types could be simplified based on formatter changes above. most drupal data structures do not make it to the front end. FieldAddress tends to be broken into good segments already, but some of the others might become:

`pdfVersion: DrupalMediaDocument` -> string
`fullText: FieldFormattedText` -> string
`contacts: NodePersonProfile[]` -> simplified PressContact type
`downloads` -> simplified download type (maybe just `{ name, url }[]`? all three look consistent as noted in the template above)